libclc/amdgcn-amdhsa/libspirv/math/tables.h

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Could you point out where this header is included? I can find some includes in `generic` directory, but I don't think this header is included there.

I can't find uses for `libclc/amdgcn-amdhsa/libspirv/math/clc_sw_binary.inc` and `libclc/amdgcn-amdhsa/libspirv/math/log_base.h` as well.

Reply:
According to your suggestion, I just checked the code. These three files are needed for the new implementation, but there are still some problems, so the new implementation has not been submitted. I will delete them later. They will appear in my next submission. Thank you for your careful inspection.